pkg/sql/testdata/telemetry/function

```diff
@@ -0,0 +1,47 @@
+feature-counters
+CREATE FUNCTION myfunc() RETURNS INT LANGUAGE plpgsql AS $$
+DECLARE
+BEGIN
+a := NULL;
+b = 7 + 7;
+END
+$$
+----
+error: pq: unimplemented: plpgsql not supported for udf
+errorcodes.0A000                      1
```

Comment:
I think we should revisit this list of output to show metrics that are interested. Namely, only show those plpgsql related? Let me know if you think why the other metrics are needed here.